server/index.js

```diff
@@ -87,6 +88,14 @@ if (Config.watchconfig) {
 	});
 }
 
+/*********************************************************
+ * Run database migrations and conversions
+ *********************************************************/
+
+if (Object.values(Config.storage || {}).includes('sqlite')) {
+	child_process.execSync('node database/migrate');
```

Comment:
`../ ` and  `{stdio: 'inherit', cwd: __dirname}` (or `path.resolve(__dirname, '..')`), no?